app/src/main/java/com/android/calculator2/widget/CalculatorWidget.java

```diff
@@ -24,22 +26,25 @@
 
 import java.text.DecimalFormatSymbols;
 
+import static com.android.calculator2.util.PlayerUtil.NUMBER_RAW_FILES;
+import static com.android.calculator2.util.PlayerUtil.RAW_FILE_EQUALS;
+
 public class CalculatorWidget extends AppWidgetProvider {
     public final static String PREFERENCE_WIDGET_PREAMBLE = "com.android.calculator2.CALC_WIDGET_VALUE_";
-    public static final String DIGIT_0 = "com.android.calculator2.0";
-    public static final String DIGIT_1 = "com.android.calculator2.1";
-    public static final String DIGIT_2 = "com.android.calculator2.2";
-    public static final String DIGIT_3 = "com.android.calculator2.3";
-    public static final String DIGIT_4 = "com.android.calculator2.4";
-    public static final String DIGIT_5 = "com.android.calculator2.5";
-    public static final String DIGIT_6 = "com.android.calculator2.6";
-    public static final String DIGIT_7 = "com.android.calculator2.7";
-    public static final String DIGIT_8 = "com.android.calculator2.8";
-    public static final String DIGIT_9 = "com.android.calculator2.9";
+    public static final String DIGIT_0 = "com.android.calculator2.digit_0";
+    public static final String DIGIT_1 = "com.android.calculator2.digit_1";
+    public static final String DIGIT_2 = "com.android.calculator2.digit_2";
+    public static final String DIGIT_3 = "com.android.calculator2.digit_3";
+    public static final String DIGIT_4 = "com.android.calculator2.digit_4";
+    public static final String DIGIT_5 = "com.android.calculator2.digit_5";
+    public static final String DIGIT_6 = "com.android.calculator2.digit_6";
+    public static final String DIGIT_7 = "com.android.calculator2.digit_7";
+    public static final String DIGIT_8 = "com.android.calculator2.digit_8";
+    public static final String DIGIT_9 = "com.android.calculator2.digit_9";
     public static final String DOT = "com.android.calculator2.dot";
     public static final String PLUS = "com.android.calculator2.plus";
     public static final String MINUS = "com.android.calculator2.minus";
-    public static final String MUL = "com.android.calculator2.mul";
+    public static final String MUL = "com.android.calculator2.multiplied_by";
```

Comment:
Maybe we should keep the original "mul" to stay as synchronized with the upstream repository as possible? I.e. rename audio files from "multiplied_by.wav" to "mul.wav".